doc/manipulating_images/manipulating_images.rst

```diff
@@ -446,85 +431,74 @@ extended to several classes, using the
 Thresholding
 ------------
 
-Voxels with better p-values are kept as voxels of interest.
-Applying a threshold to an array
-is easy thanks to numpy indexing à la Matlab.
-
-.. literalinclude:: ../../examples/04_manipulating_images/plot_roi_extraction.py
-    :start-after: # Thresholding
-    :end-before: # Binarization and intersection with VT mask
+Now, we threshold the statistical map to have better representation of voxels of
+interest. The statistical values above threshold=5 will have most intense
+voxels representing the target of regions. We visualize the same below using
```

Comment:
The statistical values above threshold=5 correspond to the most intense voxels, that  we take as target region. (If I understand correctly)